src/components/molecules/OnlineStats/OnlineStats.tsx

```diff
@@ -345,8 +347,7 @@ const OnlineStats: React.FC = () => {
         <OverlayTrigger
           trigger="click"
           placement="bottom-end"
-          overlay={popover}
-          rootClose={!selectedUserProfile} // allows modal inside popover
+          overlay={popover} // allows modal inside popover
```

Comment:
Are these changes correct? Since removing the `UserProfileModal` from here are these needed anymore?

Reply:
I can't tell, because this component OnlineStats is not used anywhere in the project. But it's OnlineStats.scss is used.

Reply:
It looks like `OnlineStats` and the corresponding `.scss` file are related to the Playa template, which is legacy code and on the way to being deleted. So it looks like any concerns related to that aren't relevant here.